libtransmission/peer-mgr.h

```diff
@@ -530,6 +550,9 @@ class tr_peer_info
     time_t connection_changed_at_ = {};
     time_t piece_data_at_ = {};
 
+    size_t bytes_sent_to_client_ = 0;
+    size_t bytes_sent_to_peer_ = 0;
```

Comment:
These 2 fields are different from `tr_peer.blocks_sent_to_client` and `tr_peer.cancels_sent_to_peer` in that they are:

1. Persistent across connect/discconects.
2. Cumulative across the entire Transmission session (as opposed to be cumulative over the last `n` seconds)

Can you help me understand why are they the way they are? If there's a good reason, then `bytesToClient` and `bytesToPeer` needs a different name to differentiate from `blocksToPeer` and `bytesToPeer`.

https://github.com/transmission/transmission/blob/ac5c9e082da257e102eb4ff18f2e433976a585d1/libtransmission/peer-common.h#L262-L266

Reply:
I tend to keep the same behavior as libtorrent, but it's fine even if it's not persistent. An external tool should be able to handle this.

Reply:
If it works for 3rd party tools, then I prefer to keep `bytesToClient` and `bytesToPeer` consistent with `blocksToPeer` and `blocksToClient`, so that it's less confusing for RPC API users. That means:

1. The stats reset as soon as the peer disconnects.
2. Cumulative over the last 60 seconds.

It should be pretty straightforward to implement, the code is 90% similar. Thanks for your work @cdowen!